thinkhazard/static/js/report.js

```diff
@@ -16,21 +19,27 @@
   });
 
   // Reload the map when a hazard type is selected
+  var hazardType;
   $('#hazard-types-list a[data-toggle="tab"]').on('shown.bs.tab', function(e) {
     var target = this.href.split('#');
     hazardType = target[1];
-    addMap();
+    addMap(hazardType);
   });
 
   // Add a new map to the page when the window changes size
+  var timeoutId = -1;
   $(window).resize(function() {
     if (timeoutId != -1) {
       clearTimeout(timeoutId);
     }
-    timeoutId = setTimeout(addMap, 500);
+    timeoutId = setTimeout(function() {
+      addMap(hazardType);
+      timeoutId = -1;
+    }, 500);
```

Comment:
Do we really need to affect a "-1" value to the timeoutId?
What if we call `clearTimeout` each time?